cmd/prune-volumes.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/fromanirh/pack8s/internal/pkg/podman"
+)
+
+// NewVolumesPruneCommand returns command to prune unused volumes
+func NewVolumesPruneCommand() *cobra.Command {
```

Comment:
I'd call PruneVolumes for (internal) consistency, but here no big deal if you keep the current name